kubelet: sysctls allows slashes as a separator (#30604)
kubernetes/website: content/en/docs/tasks/administer-cluster/sysctl-cluster.md

Kubernetes cluster using the {{< glossary_tooltip term_id="sysctl" >}}
interface.

{{< note >}}
Starting from Kubernetes version 1.23, the kubelet supports the use of either `/` or `.`
as separators for sysctl names. 
For example, you can represent the same sysctl name as `kernel.shm_rmid_forced` using a
period as the separator, or as `kernel/shm_rmid_forced` using a slash as a separator.
For more sysctl parameter conversion method details, please refer to
the page [sysctl.d(5)](https://man7.org/linux/man-pages/man5/sysctl.d.5.html) from
the Linux man-pages project.
Setting Sysctls for a Pod and PodSecurityPolicy features do not yet support 

> setting sysctls with slashes.

Reviewer: Should we mention that `Setting Sysctls for a Pod and PodSecurityPolicy features will support slashes in 1.24 and later`?

The reason should be APIServer need to be compatible with 1.n-1 version kubelet.

Reviewer: The description is correct to me. LGTM.

Author: I will update website PR after 1.24 adds this feature.

Reviewer: /lgtm

- must not allow to gain CPU or memory resources outside of the resource limits
  of a pod.

{{< note >}}

> Starting from Kubernetes version 1.23, kubelet supports the use of "/" or "." as separators for sysctl.

Reviewer 2: ```suggestion
Starting from Kubernetes version 1.23, the kubelet supports the use of either `/` or `.`
as separators for sysctl names.
```

Author: Thanks for your suggestion, I will update it.

Sysctls can be converted to a slash as a separator.

> For example, the sysctl `kernel.shm_rmid_forced` can also be written as `kernel/shm_rmid_forced` with a slash as a separator.

Reviewer 2: ```suggestion
For example, you can represent the same sysctl name as `kernel.shm_rmid_forced` using a
period as the separator, or as `kernel/shm_rmid_forced` using a slash as a separator.
```

Author: Thanks for your suggestion, I will update it.

For more sysctl parameter conversion method details, please refer to

> [Linux sysctl documentation](https://man7.org/linux/man-pages/man5/sysctl.d.5.html)

Reviewer 2: ```suggestion
the page [sysctl.d(5)](https://man7.org/linux/man-pages/man5/sysctl.d.5.html) from
the Linux man-pages project.
```

I'd consider linking to https://www.freedesktop.org/software/systemd/man/sysctl.d.html in the systemd docs instead. (I'm not sure which Linux package owns this convention).

Author: This looks more accurate, thanks.
